test/functionalTest/cases/2750_common_metrics/2750_complex_service_path.test

```diff
@@ -0,0 +1,184 @@
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Incoming Transactions and Common Metrics
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Send a GET /v2/entities with service path /A1/#
+# 02. Send a GET /v2/entities with service path /A2,/B2
+# 03. Send a GET /v2/entities with service path /#
+# 04. Send a GET /v2/entities without service path (/# is default service path)
+# 05. Ask for Metrics, see A1 and A2 as service paths, as well as two 'default service path'
+#
+
+echo "01. Send a GET /v2/entities with service path /A1/#"
+echo "==================================================="
+orionCurl --url /v2/entities --servicePath '/A1/#'
+echo
+echo
+
+
+echo "02. Send a GET /v2/entities with service path /A2,/B2"
+echo "====================================================="
+orionCurl --url /v2/entities --servicePath '/A2,/B2'
+echo
+echo
+
+
+echo "03. Send a GET /v2/entities with service path /#"
+echo "================================================"
+orionCurl --url /v2/entities --servicePath '/#'
+echo
+echo
+
+
+echo "04. Send a GET /v2/entities without service path (/# is default service path)"
+echo "============================================================================="
+orionCurl --url /v2/entities
+echo
+echo
+
+
+echo "05. Ask for Metrics, see A1 and A2 as service paths, as well as two 'default service path'"
+echo "=========================================================================================="
+orionCurl --url /admin/metrics
+echo
+echo
+
+
+--REGEXPECT--
+01. Send a GET /v2/entities with service path /A1/#
+===================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+[]
+
+
+02. Send a GET /v2/entities with service path /A2,/B2
+=====================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+[]
+
+
+03. Send a GET /v2/entities with service path /#
+================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+[]
+
+
+04. Send a GET /v2/entities without service path (/# is default service path)
+=============================================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+[]
+
+
+05. Ask for Metrics, see A1 and A2 as service paths, as well as two 'default service path'
```

Comment:
Sure, I still see it as *default* service path (as it is used if no service path is given), but it is true we have two different 'default service paths'